JiraReporter/Views/IndividualReportTemplate.cshtml

```diff
@@ -44,7 +44,7 @@ else
                                     <tbody>
                                         <tr>
                                             <td style="vertical-align:top; width:15px">
-                                                <span>&#x25cf;</span>
+                                                <span><span style="font-style:normal; font-size:25px; vertical-align:middle">&#x2022; </span></span>
```

Comment:
Create partial view for the dot. 

You will need an override for the `Partial` method on the `ExtendedTemplateBase`.